src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -28,25 +28,25 @@
  *     The caller combines the DREAM challenge-winning somatic genotyping engine of the original MuTect
  *     (<a href='http://www.nature.com/nbt/journal/v31/n3/full/nbt.2514.html'>Cibulskis et al., 2013</a>) with the
  *     assembly-based machinery of <a href="https://www.broadinstitute.org/gatk/documentation/tooldocs/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php">HaplotypeCaller</a>.
- *     Although we present the tool for somatic analyses, it may also apply to other contexts.
+ *     Although we present the tool for somatic analyses, it may also apply to other contexts, such as mitochondrial variant calling.
  * </p>
  *
  * <h3>How GATK4 Mutect2 differs from GATK3 MuTect2</h3>
  *
  * <dl>
  *     <dd>(i) The filtering functionality is now a separate tool called {@link FilterMutectCalls}.
- *     To filter further based on sequence context artifacts, additionally use {@link FilterByOrientationBias}.</dd>
+ *     To filter further based on sequence context artifacts, additionally use {@link org.broadinstitute.hellbender.tools.exome.FilterByOrientationBias}.</dd>
```

Comment:
Geraldine asked that we remove such comparisons from tool docs when we get to the official release and to place such diff info into a forum article. This differences section has been a placeholder thus far. I've created an equivalent placeholder forum doc we can reference. 

So let's remove this entire `How GATK4 Mutect2 differs from GATK3 MuTect2` section and instead say:

> For how GATK4 Mutect2 differs from GATK3 MuTect2, see https://software.broadinstitute.org/gatk/documentation/article?id=10911.

I'll take your updates here and add them to Article#10911.

Reply:
done